### PR TITLE
Ecf 111556528 user redirect

### PR DESCRIPTION
**Pull request by ericcf**
Very few changes to logic, but because it was the `ApplicationController`, almost every controller spec had to be updated.

### CODE REVIEW

File: spec/controllers/pending/participants_controller_spec.rb [hunk 1]
```diff
@@ -1,27 +1,15 @@
 require "spec_helper"
 
 describe Pending::ParticipantsController, type: :controller do
-  fixtures :all
+  fixtures :users, :participants
```
Comment (michaelwehrley) [inline spec/controllers/pending/participants_controller_spec.rb]: Why this change?

Comment (ericcf) [inline spec/controllers/pending/participants_controller_spec.rb]: I was experimenting, and it actually runs faster. I think it could make sense to do this when all you need are the people tables populated. More than that and it gets painful to manage.

File: spec/support/controller_spec_helpers.rb [hunk 2]
```diff
@@ -74,7 +74,7 @@ def sign_in_resource(resource, name)
 
 shared_examples "a rejected user action" do
   it "should redirect to the user login" do
-    expect(response).to redirect_to new_user_session_path
+    expect(response).to redirect_to new_user_session_url
```
Comment (michaelwehrley) [inline spec/support/controller_spec_helpers.rb]: What advantage are you getting here `/` vs `http://test.com/`?

**Comment by michaelwehrley** [comment]
:+1:

File: spec/support/controller_spec_helpers.rb (hunk 2 above)
Comment (ericcf) [inline spec/support/controller_spec_helpers.rb]: Interesting question. It had previously been the case that a 3xx (redirection status) from the server required an absolute URI, but after reading the most recent standard (https://tools.ietf.org/html/rfc7231) my understanding is that either absolute or relative URI's are acceptable.
